CHANGELOG.md

```diff
@@ -8,6 +8,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+## [0.7.1] - x.x.2024
+
```

Comment:
Please remove these 2 lines, it's enough to have `Unreleased` above. We will decide the date and version during the release.

Reply:
Fixed